apps/drupal/templates/layout/components--error-message.html.twig

```diff
@@ -0,0 +1,47 @@
+{#
+/**
+ * @file
+ * Theme to display an error page for missing the Component Libraries module.
+ *
+ * @see template_preprocess_page()
+ * @see html.html.twig
+ */
+#}
+
+<style>
+@import url('https://fonts.googleapis.com/css?family=Roboto+Condensed');
```

Comment:
Does this not inherit the rest of the design system's CSS?

Reply:
possibly move this to drupal-styles.scss

Reply:
The reason I put the styles inside was to make sure this template is very easy to throw away. Plus, this is likely the first thing a user is going to see on installing the theme. There's no guarantee at this step that they've webpacked the CSS assets for drupal. 

Hopefully someone only ever sees this once, installs the module and deletes this file.

Reply:
Ah, in that case go with a good ol' system font stack:

`-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji" !default;`

Reply:
Done!